routers/api/v1/repo/pull_review.go

```diff
@@ -539,3 +539,79 @@ func prepareSingleReview(ctx *context.APIContext) (*models.Review, *models.PullR
 
 	return review, pr, false
 }
+
+// DismissPullReview dismiss a review for a pull request
+func DismissPullReview(ctx *context.APIContext, opts api.DismissPullReviewOptions) {
+	// swagger:operation POST /repos/{owner}/{repo}/pulls/{index}/reviews/{id}/dismissals repository repoDismissPullReview
+	// ---
+	// summary: Dismiss a review for a pull request
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: owner
+	//   in: path
+	//   description: owner of the repo
+	//   type: string
+	//   required: true
+	// - name: repo
+	//   in: path
+	//   description: name of the repo
+	//   type: string
+	//   required: true
+	// - name: index
+	//   in: path
+	//   description: index of the pull request
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// - name: id
+	//   in: path
+	//   description: id of the review
+	//   type: integer
+	//   format: int64
+	//   required: true
+	// - name: body
+	//   in: body
+	//   required: true
+	//   schema:
+	//     "$ref": "#/definitions/DismissPullReviewOptions"
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/PullReview"
+	//   "403":
+	//     "$ref": "#/responses/forbidden"
+	//   "422":
+	//     "$ref": "#/responses/validationError"
+	if !ctx.Repo.IsAdmin() {
```

Comment:
Probably too high a permission - pull request write should probably be enough.

Reply:
I think it's necessary, Because it a manage work. which is sane with GH.

Reply:
GitHub uses target branch protection options for who can dismiss reviews. This does mean we can't cover all the possible uses cases but we can definitely add that in the future.

Reply:
:point_up: agree ...